bridgestyle/sld/fromgeostyler.py

```diff
@@ -30,10 +33,10 @@ def convert(geostyler, options=None):
     root = Element("StyledLayerDescriptor", attrib=attribs)
     namedLayer = SubElement(root, "NamedLayer")
     layerName = SubElement(namedLayer, "Name")
-    layerName.text = geostyler["name"]
+    layerName.text = _replaceSpecialCharacters('_', geostyler.get("name"))
```

Comment:
Note that for all these `.text` properties, this could be set to `None` if no `"name"` is found, which produces invalid SLD. Also, `_replaceSpecialCharacters` will fail if the property is `None`.
If we can be sure that `"name"` key exists, I'd prefer doing 
```python
_replaceSpecialCharacters('_', geostyler["name"])
```
instead. Also for the rule name below.
If the property can be `None`/missing, a default should be set (so `.get("key", "default")`, or the XML element or attribute should not be added at all imho.